d3/query-graphs.js

```diff
@@ -292,10 +309,23 @@ function abbreviateNames(treeData) {
     }, common.allChildren);
 }
 
+//
+// Escapes a string for HTML
+//
+function escapeHtml(unsafe) {
```

Comment:
I remember that this function showed up in another pull request. Can the function be moved to a common place?

Reply:
Yes, it was part of the upload-server.js
I preferred not to introduce a dependency between client code and server code.
But if you want, I can move it to a common location

Reply:
No, I think it is ok as it is. I think it is more important than the server code stays decoupled